build/wix/warningDlg.wxs

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
no, I can't. from the LICENCE:
Contributors may not remove or alter any copyright notices contained within the Program.
https://opensource.org/licenses/cpl1.0.php